### ISSUE TITLE
error during compilation Matjes-master

### ISSUE THREAD

**Issue by Soumeniisc** [bertdupe/Matjes#142]
Hi Bertdupe,

I am compiling in AMD system.  Using intel compiler. I am using your master directory code. 
I am getting an undefined reference error; details are attached in the screenshot.

CMakeFiles/Matjes.dir/src/symmetries/spglib_f08.f90.o: In function `spglib_f08_mp_spg_get_spacegroup_type_':
spglib_f08.f90:(.text+0x28): undefined reference to `spg_get_spacegroup_type'

my config.cmake:

set(CMAKE_Fortran_COMPILER ifort)
 17 set(CMAKE_CXX_COMPILER icpc)
 18 
 19 #manual adds to fortran compiler
 20 set(compadd "-mkl=parallel -march=core-avx2")                                                   #add to release and debug
 21 set(com_rel_add "")  #add to release
 22 set(com_deb_add "")                                             #add to debug 
 23 

![Screenshot from 2023-11-08 15-26-29](https://github.com/bertdupe/Matjes/assets/4995258/0a0f8c32-8ebc-42da-9da5-9707a40ebe80)

**Comment by bertdupe** [comment]
Hello,

Could you please send me your config.cmake file?
Could you please give me some detail on your computer? It AMD (should work on AMD so no problem here).
- are MKL installed?
- SPGLIB is a third party library, is it installed? Otherwise, you should compile without it - I will see in you config.cmake file

Cheers,
Bertrand

**Comment by Soumeniisc** [comment]
non commented  part of the config.cmake  is given in the previous conversation as a text. 

**System Information:**

Architecture:        x86_64
CPU op-mode(s):      32-bit, 64-bit
Byte Order:          Little Endian
CPU(s):              4
On-line CPU(s) list: 0-3
Thread(s) per core:  1
Core(s) per socket:  2
Socket(s):           2
NUMA node(s):        1
Vendor ID:           AuthenticAMD
CPU family:          25
Model:               1
Model name:          AMD EPYC 7713 64-Core Processor
Stepping:            1
CPU MHz:             1996.275
BogoMIPS:            3992.51
Hypervisor vendor:   Microsoft
Virtualization type: full
L1d cache:           32K
L1i cache:           32K
L2 cache:            512K
L3 cache:            262144K
NUMA node0 CPU(s):   0-3

**MKL installed**

probably SPGLIB not installed.

**Comment by Soumeniisc** [comment]
Hi Bertrand,

I managed to specify all the library paths and installed this version.  I am getting some I/O error for calculation with "symmetries.in". I am wondering whether it is a bug or I am making some mistakes. I have attached the screen sort of error part and symmetries.in file
![Screenshot from 2023-11-13 16-04-41](https://github.com/bertdupe/Matjes/assets/4995258/a4993e56-4f86-42e1-b826-45a68cdde1c3)
![Screenshot from 2023-11-13 16-01-52](https://github.com/bertdupe/Matjes/assets/4995258/6e83fdd7-d58e-4e91-b1f2-18977e4892f3)
